onert-micro/helpers/micro_test.h

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ONERT_MICRO_HELPERS_TEST_H
+#define ONERT_MICRO_HELPERS_TEST_H
+#include <mbed.h>
```

Comment:
This test is only enabled for mbed os target ?

Reply:
As I understand, yes

Reply:
But actually I don't find any mbed specific in this micro test helpers, so @SlavikMIPT,  maybe remove this header?